packages/vendure-plugin-sales-per-variant/README.md

```diff
@@ -0,0 +1,125 @@
+# Vendure Sales Per Variant plugin
+
+![Vendure version](https://img.shields.io/npm/dependency-version/vendure-plugin-metrics/dev/@vendure/core)
+
+### [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-metrics)
```

Comment:
```suggestion
```
This can be removed. I'm working on a new version of the plugins website, but not there yet